cmds/import_image/import_image_cmd.go

```diff
@@ -84,5 +114,28 @@ func (cmd *ImportImageCmd) Run() error {
 	cmd.Id = vgbdtgObject.Id
 	cmd.Uuid = vgbdtgObject.GlobalIdentifier
 
+	if cmd.Public {
+		execStmtRetryable := boshretry.NewRetryable(
+			func() (bool, error) {
+				id, err := vgbdtgService.CreatePublicArchiveTransaction(cmd.Id, cmd.PublicName, cmd.PublicNote, cmd.PublicNote, locations)
+				if err != nil {
+					return true, errors.New(fmt.Sprintf("There would be an active transaction in progress."))
+				}
+
+				cmd.Id = id
```

Comment:
The CreatePublicArchiveTransaction function only returns id of the image, no uuid. Actually, eCPI only uses image id to deploy VMs, so uuid is not required, if we want to also add uuid in the light stemcell for public images, we need Softlayer-go to support a new API [getGlobalIdentifier](http://sldn.softlayer.com/reference/services/SoftLayer_Virtual_Guest_Block_Device_Template_Group/getGlobalIdentifier).